src/types.cairo

```diff
@@ -1,9 +1,10 @@
-use ekubo::interfaces::core::SwapParameters;
-use ekubo::types::delta::Delta;
-use ekubo::types::keys::PoolKey;
+// use ekubo::interfaces::core::SwapParameters;
+// use ekubo::types::delta::Delta;
+// use ekubo::types::keys::PoolKey;
+use ekubo::{interfaces::core::SwapParameters, types::{delta::Delta, keys::PoolKey}};
 use starknet::ContractAddress;
 
-pub type TokenPrice = u256;
+pub type TokenPrice = u128;
 pub type TokenAmount = u256;
 pub type Decimals = u64;
```

Comment:
You don't actually store Decimals in this data type, you store 10**x where x is the decimal count, which is unclear